### PR TITLE
feat: adds text for external install

### PR DESCRIPTION
**Pull request by scefali**

### CODE REVIEW

**Comment by souredoutlook** [comment]
Stumbled on to this one by chance. This PR doesn't describe the behaviour of this URL for unpublished apps. It looks like unpublished apps will either fail to load (for general users), or for the creators show a broken install page. cc @masinette 

Ticket with a potential integrator running into issues with this link: https://sentry.zendesk.com/agent/tickets/85677

**Comment by scefali** [comment]
@souredoutlook I still need to fix a bug related to the dropdown in the UI. It should work fine for unpublished apps as long as you're on the org that created the app.

File: src/docs/product/integrations/integration-platform/public-integration.mdx [hunk 1]
```diff
@@ -57,6 +57,8 @@ def setup():
     return redirect('https://sentry.io/settings/')
 ```
 
+All public integrations can be installed on a fixed external url that you can expose to your users. That url is: `https://sentry.io/sentry-apps/<your-integration-slug>/external-install/`. For example, here is the one for `Linear`: https://sentry.io/sentry-apps/linear/external-install/. If a user has multiple organization they will be asked to pick an organization for the installation in a dropdown.
+
```
Comment (lizokm) [inline src/docs/product/integrations/integration-platform/public-integration.mdx:61]:
```suggestion
All public integrations can be installed via a fixed external url: `https://sentry.io/sentry-apps/<your-integration-slug>/external-install/`, that you can expose to users. Check out this example for `Linear`: https://sentry.io/sentry-apps/linear/external-install/. Note, if a user has multiple organizations, they'll be asked to choose the one they want for the installation from a dropdown menu.

```

**Review by lizokm — APPROVED** [review]
A few language suggestions.

**Pull request merged**